analysis2_manuscript.tex

```diff
@@ -1073,6 +1078,7 @@ \section*{Methods}
     Each of these simulations generated 100 diploid genomes.
     The configuration file used in combination with our \snakemake workflow
     for these simulations can be found at \url{https://github.com/popsim-consortium/analysis2/blob/main/workflows/config/snakemake/production_config_HomSap.yml}.
+    The running times of these simulations depend on chromosome length, the demographic model, and incorporation of selection (Figure \ref{fig:sim-runtime}).
```

Comment:
looks like you moved the reference to the runtime figure here-- we should probably include the revpoint tag too.
```suggestion
    The running times of these simulations depend on chromosome length, the demographic model, and incorporation of selection (Figure \ref{fig:sim-runtime}). \revpoint{1}{4}
```